app/components/Resource/Resource.js

```diff
@@ -27,21 +27,50 @@ class Resource extends React.Component {
 
 	render() {
 		return (
-			<div className="resource_container">
-				<h1>{this.state.resource.name}</h1>
-				<hr />
-                <CommunicationBoxes />
-                <Services description={this.state.resource.long_description} services={this.state.resource.services}/>
-				<div className="infocontainer">
-					<AddressInfo addresses={this.state.resource.addresses} />
-					<BusinessHours schedule_days={this.state.resource.addresses} />
-					<PhoneNumber phones={this.state.resource.phones} />
-					<Website website={this.state.resource.website} />
-					<Languages />
-					<hr />
-					<div className="resource_editslabel"><p>Make Edits</p></div>
-				</div>
-			</div>
+      <div className="org-container">
+        <article className="org">
+          <div className="main-column">
+            <section className="org-summary">
+              <header>
+                <img className="org-img" src="http://lorempixel.com/100/100/city/" />
+                <div className="org-info">
+                  <h1>{this.state.resource.name}</h1>
+                  <h4>Category 1, Category 2, Category 3</h4>
+                  <p><AddressInfo addresses={this.state.resource.addresses} /></p>
+                </div>
+              </header>
+              <div className="rating-summary-container">
+                <div className="rating-summary excellent">
+                  <i className="material-icons">sentiment_very_satisfied</i>
+                  <span>{Math.floor(Math.random()*10)%6}/5</span>
+                </div>
+                <div className="rating-summary-details">
+                  <a href="#reviews">12 reviews</a>
+                </div>
+              </div>
+              <div className="org-desc">
+                <p>This is an organisation's long description{this.props.description}</p>
```

Comment:
Hashes are placeholders so you can easily find empty links. Should have
mentioned that. And the resource description is also placeholder.

On Wednesday, 29 June 2016, James Hamlin <notifications@github.com> wrote:

> In app/components/Resource/Resource.js
> <https://github.com/ShelterTechSF/askdarcel-web/pull/62#discussion_r69075316>
> :
>
> > +                  <h1>{this.state.resource.name}</h1>
> > +                  <h4>Category 1, Category 2, Category 3</h4>
> > +                  <p><AddressInfo addresses={this.state.resource.addresses} /></p>
> > +                </div>
> > +              </header>
> > +              <div className="rating-summary-container">
> > +                <div className="rating-summary excellent">
> > +                  <i className="material-icons">sentiment_very_satisfied</i>
> > +                  <span>{Math.floor(Math.random()*10)%6}/5</span>
> > +                </div>
> > +                <div className="rating-summary-details">
> > +                  <a href="#reviews">12 reviews</a>
> > +                </div>
> > +              </div>
> > +              <div className="org-desc">
> > +                <p>This is an organisation's long description{this.props.description}</p>
>
> Ah, I still see some expressions scattered around, so I thought the
> correct one was intended to be in here. If not, I'm OK with proceeding with
> this too.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/ShelterTechSF/askdarcel-web/pull/62/files/6e4ed978db9f126d8fe4d00a168f17d3ff46eef6#r69075316>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ACmL3zk_ZVhV43VaVQXyj66gfvQ1MaO8ks5qQ1rRgaJpZM4I-fT6>
> .
>


-- 
Derek Fidler
Product Design, Bebo

Follow me on Twitter @derekfidler <http://twitter.com/derekfidler>
www.derekfidler.com
df@derekfidler.com <derek@derekfidler.com>
415.857.0879